### PR TITLE
Add the AnimatorJs library

### PR DESCRIPTION
**Pull request by RandomGamingDev**
A javascript animation viewer based on a custom animation API primarily designed for p5.js and allows for dynamic, lightweight, real-time rendered browser animations controlled purely by code.

The library's repository: https://github.com/RandomGamingDev/AnimatorJs/tree/main

### CODE REVIEW

**Comment by davepagurek** [comment]
Thanks for bringing these libraries over to this repo! I think since moving from the old repo, we changed the image format a bit, so now `featuredImage` is the path to the image and a separate top-level `featuredImageAlt` should contain the alt text:
image

**Comment by RandomGamingDev** [comment]
Sorry about the late response, I've updated the library's yaml to fit the new format as specified by @davepagurek in this message: https://github.com/processing/p5.js-website/pull/626#issuecomment-2457129685

File: src/content/libraries/en/AnimatorJs.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: AnimatorJs
+description: A javascript animation viewer based on a custom animation API primarily designed for p5.js and allows for dynamic, lightweight, real-time rendered browser animations controlled purely by code.
+category: drawing
+sourceUrl: https://github.com/RandomGamingDev/AnimatorJs/
+featuredImage: AnimatorJs.png
```
Comment (davepagurek) [inline src/content/libraries/en/AnimatorJs.yaml]:
```suggestion
featuredImage: "../AnimatorJs.png"
```

File: src/content/libraries/en/AnimatorJs.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: AnimatorJs
+description: A javascript animation viewer based on a custom animation API primarily designed for p5.js and allows for dynamic, lightweight, real-time rendered browser animations controlled purely by code.
+category: drawing
+sourceUrl: https://github.com/RandomGamingDev/AnimatorJs/
+featuredImage: "../AnimatorJs.png"
```
Comment (davepagurek) [inline src/content/libraries/en/AnimatorJs.yaml]:
```suggestion
featuredImage: "../images/AnimatorJs.png"
```

**Pull request merged**